custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -685,9 +685,16 @@ def fit(self, X, y, output_dir, class_order=None, row_weights=None, parameters=N
                         parameters=parameters,
                     )
 
-                    self._custom_task_class_instance.save(self._model_dir)
+                    try:
+                        self._custom_task_class_instance.save(self._model_dir)
+                    except Exception:
+                        raise DrumCommonException(
+                            "There appears to be an issue with your save hook in custom.py"
```

Comment:
Would change this to be `An error occurred when saving your custom task. Ensure all variables stored on self are pickleable or excluded when saving`